images/macos/provision/core/commonutils.sh

```diff
@@ -13,6 +13,9 @@ for package in $cask_packages; do
     brew install --cask $package
 done
 
+# Execute AppleScript to change security preferences for virtualbox
```

Comment:
Please add an additional info about  menu we use to allow this kext modules.

Reply:
Done